lib/Authentication/AuthTokens/ShibAuthToken.php

```diff
@@ -204,4 +204,22 @@ public static function isStateless() {
         return true;
     }
 
+    private function getEntitlementErrorMessage()
+    {
+        $refedsResAndSchURL = "https://refeds.org/category/research-and-scholarship";
+        $resourceLink = "https://docs.egi.eu/internal/configuration-database/access";
+        $sectionFragmentInfo = "/#using-institutional-account-via-egi-check-in";
+        $documentationURL = $resourceLink . $sectionFragmentInfo;
```

Comment:
codeclimate is complaining about the length of these lines - I don't see a way to split them up that makes them anymore readable.

I'll mark them as `wontfix`.